model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -51,7 +51,7 @@ class InsertReverseChannels(BackReplacementPattern):
     enabled = False
 
     @staticmethod
-    def get_fw_index(node: Node, idx: int):
+    def get_fw_index(node: Node, idx: np.ndarray) -> np.ndarray:
```

Comment:
I see no reason to change `int` -> `np.ndarray`. In all callers we pass scalar int.

Reply:
Changed to int.